sdks/apis.do/README.md

```diff
@@ -68,60 +68,145 @@ APIs.do provides unified access to the entire `.do` ecosystem:
 ## 📋 Quick Start
 
 ```typescript
-import { ApiClient } from 'apis.do'
+import { API } from 'apis.do'
 
 // Initialize the client
-const api = new ApiClient({
+const api = new API({
   apiKey: 'your-api-key', // Optional: API key for authenticated requests
 })
 
 // Basic CRUD operations
 const getUsers = async () => {
-  const users = await api.list('users')
-  console.log(users.data)
+  const users = await api.users.find()
+  console.log(users)
 }
 
 const getUserById = async (id: string) => {
-  const user = await api.getById('users', id)
+  const user = await api.users.get(id)
   console.log(user)
 }
 
 const createUser = async (userData: any) => {
-  const newUser = await api.create('users', userData)
+  const newUser = await api.users.create(userData)
   console.log(newUser)
 }
 
 const updateUser = async (id: string, userData: any) => {
-  const updatedUser = await api.update('users', id, userData)
+  const updatedUser = await api.users.update(id, userData)
   console.log(updatedUser)
 }
 
 const deleteUser = async (id: string) => {
-  await api.remove('users', id)
+  await api.users.delete(id)
   console.log('User deleted')
 }
 ```
 
+## 🧩 Collection Types
+
+The APIs.do SDK provides access to all collections in the platform. Here are the main collection types available:
+
+### AI Collections
+- `functions` - AI functions with type, schema, and code/prompt
+- `workflows` - Declarative state machines for orchestration
+- `agents` - Autonomous digital workers
+
+### Data Collections
+- `things` - Core data entities with properties
+- `nouns` - Categories or types of Things
+- `verbs` - Action forms and relationships
+
+### Event Collections
+- `triggers` - Events that initiate workflows
+- `searches` - Query operations for retrieving data
+- `actions` - Tasks performed within workflows
+
+### Observability Collections
+- `generations` - Records of system state before/after an Action
+- `events` - System events with timestamps and metadata
+- `traces` - Execution traces for debugging
+
+### Integration Collections
+- `integrations` - External system connections
+- `integration-triggers` - Events from external systems
+- `integration-actions` - Operations on external systems
+
+## 📊 Strongly-Typed Collection Examples
+
+```typescript
+import { API } from 'apis.do'
+import type { Function, Workflow, Agent, Thing, Action, Generation } from 'apis.do/types'
+
+const api = new API({
+  apiKey: process.env.APIS_DO_API_KEY || process.env.DO_API_KEY
+})
+
+// Working with AI Functions
+async function getFunctions() {
+  const functions = await api.functions.find()
+  return functions
+}
+
+async function createFunction(functionData: Partial<Function>) {
+  return await api.functions.create({
+    name: 'Example Function',
+    type: 'Generation',
+    format: 'Object',
+    schema: {
+      type: 'object',
```

Comment:
the schema should be our descriptive schema format like:

{
  productType: 'App | API | Marketplace | Platform | Packaged Service | Professional Service | Website',
  customer: 'ideal customer profile in 3-5 words',
  solution: 'describe the offer in 4-10 words',
  description: 'website meta description',
}